projects/extension/src/background/index.ts

```diff
@@ -28,14 +28,21 @@ export interface Background extends Window {
   }
 }
 
+type logStructure = {
```

Comment:
Why not an interface?

```suggestion
interface LogStructure = {
```

Reply:
I use type on a structure when its used inside an interface - but in latest versions of TS type and interface tend to be the same so it does not make much of difference